init_project.sh

```diff
@@ -39,9 +39,31 @@ VPIPE_DIR=$(dirname "$0")
 
 cp -iv "$VPIPE_DIR/vpipe.config" "$PROJECT_DIR/"
 
+# guess activation command
+ACTIVATE=
+# search order:
+# - 'V-pipe' conda environment
+# - base conda environment
+for VPIPEENV in "$VPIPE_DIR/../miniconda3"{/envs/V-pipe,}; do
+	if [ -d "${VPIPEENV}" ] && [ -x "${VPIPEENV}/bin/activate" ] && [ -x "${VPIPEENV}/bin/conda" ] && [ -x "${VPIPEENV}/bin/snakemake" ]; then
+		echo "Conda environment found in ${VPIPEENV}"
+		if [ -x "${VPIPEENV}/envs/V-pipe/bin/snakemake" ]; then
+			ACTIVATE=". ${VPIPEENV}/bin/activate 'V-pipe'"
+		else
+			ACTIVATE=". ${VPIPEENV}/bin/activate 'base'"
+		fi
+		break
+	fi
+done
+if [ -z "${ACTIVATE}" ]; then
+	echo "Warning: cannot detect conda environment" 1>&2
+fi
+
+
 cat > "$PROJECT_DIR/vpipe" <<EOF
 #!/bin/sh
-snakemake -s "$VPIPE_DIR/vpipe.snake" "\$@"
+${ACTIVATE}
```

Comment:
Note that the `init_project.sh` script is supposed to also work in installations managed by advanced users not using our quick-install script.

Silently activating some environment is likely to break the workflows of advanced users, including myself, because having a `miniconda3` directory in one's home directory is not uncommon.

Also, `--use-conda` may not be the desired choice for many advanced users. And without knowing that it is there, they might waste a lot of time trying to figure out why things do not work.

To address these issues, please amend the behavior as follows:
- Add a `--no-conda`  option. If that is passed, `ACTIVATE` will not be set and no conda environments will be activated. Also, the `--use-conda` option shall not be passed in this case.
- If `--no-conda` is not passed, before trying to automagically discover the environment to activate, check if any conda environment (including `base`!) is currently active. If not, proceed as proposed in the PR. If yes, emit a warning that the currently-active conda environment will be used and leave `ACTIVATE` unset.

Reply:
I am considering that advanced users are able to call `snakemake` on their own. I am really trying to fool proof it for the beginner users.

To you want support for `--no-conda` right now before current release cycle?

Alternative possibility: 
- option `-n` in `init_project.sh` that suppresses all the conda additions.

Reply:
Yes, I think this is the minimum. Please do it (-n).